apps/judicial-system/web/src/routes/Court/Indictments/Summary/Summary.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useContext, useState } from 'react'
+import { useIntl } from 'react-intl'
+import router from 'next/router'
+
+import { Box, Text } from '@island.is/island-ui/core'
+import * as constants from '@island.is/judicial-system/consts'
+import { core } from '@island.is/judicial-system-web/messages'
+import {
+  FormContentContainer,
+  FormContext,
+  FormFooter,
+  InfoCardClosedIndictment,
+  Modal,
+  PageHeader,
+  PageLayout,
+  PageTitle,
+  SectionHeading,
+} from '@island.is/judicial-system-web/src/components'
+import {
+  Defendants,
+  Prosecutor,
+} from '@island.is/judicial-system-web/src/components/CaseInfo/CaseInfo'
+import { RenderFiles } from '@island.is/judicial-system-web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList'
```

Comment:
Consider importing from `@island.is/judicial-system-web/src/components` for consistency.